.github/workflows/ci.yaml

```diff
@@ -1,4 +1,4 @@
-name: GitHub Actions CI
+name: CI - Main
```

Comment:
Isn't so good, IMO.
But I will not require any changes in the hope of subsequent changes proposed by @xavier2k6, so that it will be divided into several clearer names, such as `CI - Windows builds` etc.

Reply:
OK, will leave it to others.